### DISCUSSION TITLE
Failed to show preview in README

### DISCUSSION THREAD

**Discussion by haunt98** [community/community#16966] [Mobile] [answered]
Failed to show preview in README

![Screenshot_2022-05-19-12-21-15-778_com.github.android.jpg](https://user-images.githubusercontent.com/5424272/169454841-fbaa139a-5cfd-4300-ac80-fb5a34388477.jpg)

https://github.com/TheRobotFactory/EightBit-Atari-Fonts

**Comment by mxie** [comment, marked as answer]
Thanks for reporting this @haunt98! I'll relay this back to our team.
